client/my-sites/site-settings/composing/style.scss

```diff
@@ -0,0 +1,7 @@
+.composing__module-settings.is-indented {
+	margin: 16px 32px 0;
```

Comment:
Yes, thanks for pointing that out. I've already added that to the description of the PR, and I'm planning to address it after it's merged.